internal/metamorph/server.go

```diff
@@ -334,6 +334,41 @@ func (s *Server) GetTransaction(ctx context.Context, req *metamorph_api.Transact
 	return txn, nil
 }
 
+func (s *Server) GetManyTransactions(ctx context.Context, req *metamorph_api.ManyTransactionsStatusRequest) (*metamorph_api.Transactions, error) {
```

Comment:
This function has no test coverage. Could you add a unit test to `server_test.go` for this function?

Reply:
Added new test